src/Microsoft.Identity.Web/CertificateManagement/DefaultCertificateLoader.cs

```diff
@@ -116,6 +116,11 @@ private static X509Certificate2 LoadFromBase64Encoded(string certificateBase64,
 
             KeyVaultCertificateWithPolicy certificate = certificateClient.GetCertificate(certificateName);
 
+            if (certificate.Properties.NotBefore <= DateTime.UtcNow && certificate.Properties.ExpiresOn <= DateTime.UtcNow)
```

Comment:
A few comments: 

1. NotBefore and ExpiresOn can be null. Not sure why, but I think we should treat that case separately
2. nit; I'm always confused about mixing DateTimeOffset and DateTime.UtcNow. Probably fine. Might be cleaner to use DateTimeOffset.UtcNow
3. I think the condition to accept the cert is : 

NotBefore ---------- now ------------- ExpOn

so the condition to reject the cert is: 

now < NotBefore OR now > ExpOn